azure-communication-ui/build.gradle

```diff
@@ -46,10 +46,10 @@ buildscript {
             // and enable the beta features
 
             // Latest GA version of the SDK
-            azure_calling_sdk_version_ga = '2.7.0'
+            azure_calling_sdk_version_ga = '2.9.0-beta.1'
```

Comment:
I'd encourage setting these correctly and not putting Beta here.

If beta is required for features, ideally use flagging if at all possible so that we can continue to ship on latest GA at any moment.

I do have some compassion because these features are targeted for release, and started prior to flags being widely, but at some point in the future, I'll be much more strict about Beta api's being used in the GA line.

Reply:
Yes Adam, currently merging to my feature branch only. Expected GA is around next week for TelecomManager SDK. Before merging to develop if not in GA, will add feature flag.